linter_exclusions.yml

```
@@ -3624,6 +3624,9 @@ vmss create:
     automatic_repairs_grace_period:
       rule_exclusions:
       - option_length_too_long
+    automatic_repairs_action:
+      rule_exclusions:
+      - option_length_too_long
```

Could we consider adding a short alias for this new parameter?

```suggestion
    automatic_repairs_action:
      rule_exclusions:
      - option_length_too_long
```

---

This parameter is consistent with the style of `automatic_repairs_grace_period`, I don’t know if it needs to be changed together